docs/content/v2.20/drivers-orms/java/yugabyte-jdbc.md

```diff
@@ -94,12 +94,55 @@ Install the added dependency using `mvn install`.
 If you are using [Gradle](https://docs.gradle.org/current/samples/sample_building_java_applications.html), add the following dependencies to your `build.gradle` file:
 
 ```java
-implementation 'com.yugabyte:jdbc-yugabytedb:42.3.5-yb-8'
+implementation 'com.yugabyte:jdbc-yugabytedb:42.7.3-yb-1'
 implementation 'com.zaxxer:HikariCP:4.0.3'
 ```
 
 ### Step 2: Set up the database connection
 
+<table>
```

Comment:
Can we not just xref to the driver page for this

Reply:
This was inadvertent, removed